.dir-locals.el

```diff
@@ -0,0 +1 @@
+((erlang-mode . ((indent-tabs-mode . t))))
```

Comment:
I see that at least in testnet some files have only spaces e.g. https://github.com/aeternity/testnet/blob/054824a8ff56b517bd2a86fd90c9c5a0fe030e5b/apps/ae_core/src/consensus/download_blocks.erl while others have mix of tabs and spaces.